src/services/services.ts

```diff
@@ -345,6 +346,88 @@ namespace ts {
         }
     }
 
+    class TokenOrIdentifierObject implements Token {
+        public kind: SyntaxKind;
+        public flags: NodeFlags;
+        public parent: Node;
+        public jsDocComments: JSDocComment[];
+        public __tokenTag: any;
+
+        constructor(public pos: number, public end: number) {

```

Comment:
can you list all assignments to properties in the same order as in the NodeObject ?